src/database/hooks/useSaveAnonChatHook.ts

```diff
@@ -37,8 +37,8 @@ const useSaveAnonChatHook = () => {
     type: string
   ) => {
     if (!localDb) return;
-
     const chatName = await getAnonymousChatName(object?.members);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `getAnonymousChatName` function is asynchronous but it's not being handled properly. It should be wrapped in a try-catch block to handle any potential errors and avoid unhandled promise rejections.
```diff
- const chatName = await getAnonymousChatName(object?.members);
+ try {
+   const chatName = await getAnonymousChatName(object?.members);
+ } catch (error) {
+   console.error('Error getting anonymous chat name:', error);
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->